cloudquery-cloud/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "727e53a9-0bbe-4277-8ed0-9e9425fe34de",
+  "app_id": "cloudquery-sync",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "Simple, Fast and Extensible Data Movement",
+    "title": "CloudQuery Cloud",
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Cost Management",
+      "Category::Data Stores",
+      "Category::Developer Tools",
+      "Category::Security",
+      "Offering::Integration",
+      "Category::Cloud"
```

Comment:
```suggestion
      "Category::Developer Tools",
      "Offering::Integration",
      "Category::Cloud"
```
Removing inapplicable categories. Please also add specific data types you're querying. You can reference the available list [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags). This helps customers quickly determine what the integration does and what data types it manipulates.
<img width="528" alt="Screenshot 2024-10-09 at 10 24 18 AM" src="https://github.com/user-attachments/assets/f84e9012-3073-4598-a78d-ae1cc2552280">

Reply:
The extension only extracts data from DataDog to a user configured database/datastore, and it can be used in/for any of the previously listed categories.

Reply:
I've updated my suggestion then.

Reply:
Thank you, updated